asyncio_mqtt/client.py

```diff
@@ -233,7 +233,7 @@ def __init__(  # noqa: C901
         *,
         username: str | None = None,
         password: str | None = None,
-        logger: logging.Logger | None = None,
+        logger: logging.Logger = MQTT_LOGGER,
```

Comment:
Prefer to use `None` for default values. It makes it easier to extend `Client` (e.g., through inheritance).

In turn, assign the actual default value inside the function itself. E.g.:

```python
if logger is None:
    logger = MQTT_LOGGER
self._logger = logger
self._client.enable_logger(logger)
```

Reply:
Done.